common/types/block_state.go

```diff
@@ -233,6 +234,44 @@ func (bs *StateBlockList) Deserialize(text []byte) error {
 	return nil
 }
 
+func (b *StateBlock) TableSchema() string {
+	//sb := new(StateBlock)
+	//ts := TableSchema{}
+	//ts.Column = append(ts.Column, sb.GetHash())
+	//ts.Column = append(ts.Column, sb.Type)
+	//ts.Column = append(ts.Column, sb.Address)
+	//ts.Column = append(ts.Column, sb.Timestamp)
+	//return ts
+	str := fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s
+		(   id integer PRIMARY KEY AUTOINCREMENT,
+			hash %s,
+			type %s,
+			address %s,
+			timestamp %s
+		)`,
+		b.TableName(),
+		ConvertSchemaType(b.GetHash()),
+		ConvertSchemaType(b.Type),
+		ConvertSchemaType(b.Address),
+		ConvertSchemaType(b.Timestamp))
```

Comment:
How to deal with different databases?

Reply:
i add a parameter to distinguish database， i will test mysql later